install/linux/usr/share/odemis/hwtest/piezomotor.odm.yaml

```diff
@@ -0,0 +1,30 @@
+MultiBeamSEM: {
+  class: Microscope,
+  role: mbsem,
```

Comment:
Isn't that a little presumptuous for a microscope with just a stage? ;-) Let's use just "brightfield" as role.